docs/developer-docs/latest/setup-deployment-guides/installation/cli.md

```diff
@@ -1,34 +1,64 @@
 ---
-title: Install from CLI - Strapi Developer Documentation
+title: Installing from CLI - Strapi Developer Documentation
 description: Fast-track local install for getting Strapi running on your computer in less than a minute.
 ---
 
-# Installing from CLI
-
-Fast-track local install for getting Strapi running on your computer.
-
-[[toc]]
-
-## Step 1: Make sure requirements are met
+<style lang="scss" scoped>
+/*
+    We override the :::warning and :::danger callouts for specific uses here.
+    The CSS is scoped so this won't affect the rest of the docs.
+
+    Eventually this will be turned into custom blocks or VuePress components,
+    once I understand better how markdown-it and markdown-it-custom-block work.
+  */
+  .custom-block.congrats,
+  .custom-block.danger {
+    border-left-width: .25rem;
+  }
+
+  .custom-block.danger {
+    margin-top: 2em;
+    margin-bottom: 2em;
+
+    .custom-block-title, p, li {
+      color: rgb(44, 62, 80);
+    }
+    a {
+      color: #007eff;
+    }
+  }
+
+  .custom-block.danger {
+    background-color: rgba(129,107,250, .05);
+    border-color: rgb(129,107,250);
+
+    .custom-block-title {
+      color: rgb(129,107,250);
+      font-weight: bold;
+    }
+  }
+
+  .custom-block.details {
+      color: rgb(44, 62, 80);
+  }
+</style>
 
-#### Node.js
-
-Strapi only requires [Node.js](https://nodejs.org). Strapi only supports **LTS versions** of Node.js, the current recommended version to run Strapi is **Node LTS v14**. Other versions of Node.js may not be compatible with the latest release of Strapi.
+# Installing from CLI
 
-This is everything you need to run Strapi on your local environment.
+Strapi CLI (Command Line Interface) installation scripts are the fastest way to get Strapi running locally. The following guide is the installation option most recommended by Strapi.
 
-| Software | Minimum version | Recommended version |
-| -------- | --------------- | ------------------- |
-| Node.js  | 12.x            | 14.x                |
-| npm      | 6.x             | 6.x                 |
+## Preparing the installation
 
-#### Yarn (optional)
+The CLI installation guide requires 2 softwares to be already installed on your computer:
```

Comment:
```suggestion
The CLI installation guide requires at least two software prerequisites to be already installed on your computer:
```

Reply:
Here I recommend more, "The CLI installation guide has at least two software prerequisite for your environment"